### ISSUE TITLE
Support of L4T R32.5.1 ?

### ISSUE THREAD

**Issue by amcsSH** [JetsonHacksNano/rootOnUSB#21]
Dear,

I use the latest version of rootOnUSB and follow the README and Videos but after I reboot the system to start from USB drive, the starts fail because no /dev/sda was found.

I use an SSD installed in a Geekworm Jetson Nano T300 expansion adapter with Jetson L4T R32.5.1 .

Any suggestions or hints?

Thanks in advance, Andreas

**Comment by xiaohai-AI** [comment]
Yes, I also runing L4TR32.5.1. After reboot, the nano device seems even locked at the BIG NVIDIA LOGO, and then exit and LOGO again until for ever.
since this Repo alardy two years ago, may be need updated.
Just want to know if I install L4T R32.2.4  get this external SSD work first , then upgrade to L4T R32.5.1.
Is this a eay way or not ?

Thanks

Dan

**Comment by xiaohai-AI** [comment]
Hi there

I just find a solution on the net, and I already successfully got the job done  yesterday!!
you can following the detail instruction there!

https://robroyce.dev/2021/02/11/how-to-boot-jetson-nano-from-usb-3-0-natively-with-jetpack-4-5/


Dan

**Comment by jantielens** [comment]
Thanks for sharing, excellent timing. :)
